src/prefect_server/graphql/schema/runs.graphql

```diff
@@ -44,6 +44,9 @@ extend type Mutation {
   "Delete a flow run."
   delete_flow_run(input: delete_flow_run_input!): success_payload
 
+  "Delete a group of flow runs."
+  delete_flow_runs(input: delete_flow_runs_input!): deleted_count_payload
+
```

Comment:
Generally, does it make sense to implement bulk operations by extending the original interface or adding a new route?